client/modules/reducer.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from 'redux';
```

Comment:
This is great. Somehow I feel `reducer` is more special than the other files here. 
A couple of ideas:
1. Rename it to index.js?
2. Other reducers become folders?

Thoughts?